utils/core-modules/contracts/interfaces/IAssociatedSystemsModule.sol

```diff
@@ -1,9 +1,42 @@
 //SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-/// @title Allows for the registration and tracking of auxillery contracts which also follow the proxy architecture
+/**
+ * @title Module for connecting a system with other associated systems.
+ *
+ * Associated systems can me managed or unmanaged.
+ *
+ * Managed systems are connected via a proxy, which means that their implementation can be updated, and the system controls the execution context of the associated system. Example, an snxUSD token connected to the system, and controlled by the system.
+ *
+ * Unmanaged systems are just addresses tracked by the system, for which it has no control whatsoever. Example, Uniswap v3, Curve, etc.
+ *
+ * Associated systems, managed or unmanaged, become available to all system modules for communication and interaction, but as opposed to inter-modular communications, associated system interactions will require the use of `CALL`.
+ */
```

Comment:
Maybe explain the concepts of 'kinds' of systems up here? (Token, Nft, maybe we add a 'generic' one?)

Reply:
Updated the docs, please review.
I don't think we need a generic new type, `KIND_UNMANAGED` takes care of that.